tests/unittests/BackendTestUtils.cpp

```diff
@@ -700,6 +700,7 @@ void inferConvDKKC8(Tensor *out, llvm::StringRef kind) {
   auto *filter = mod.createPlaceholder(ElemKind::FloatTy, {192, 3, 3, 32},
                                        "filter", false);
   auto *filterTensor = bindings.allocate(filter);
+  filterTensor->zero();
```

Comment:
So here the line below looks wrong:
`          FH.at({i, j, k, k}) = (i + j + k + l) / 200.0;
`
Should that last `k` be an `l` ?

If I remove the zeroing and change it to `l` the test fails, and it's not obvious why.

@nadavrot I think you wrote this one, do you remember the details?